Oryx.Cognite/src/Handler.fs

```diff
@@ -91,7 +91,14 @@ module Handler =
 
     /// Decode response message into a ResponseException.
     let decodeError (response: HttpResponseMessage) : Task<HandlerError<ResponseException>> = task {
-        if response.Content.Headers.ContentType.MediaType.Contains "application/json" then
+        let mediaType =
+           Option.ofObj response.Content
+           |> Option.bind (fun content -> content.Headers |> Option.ofObj)
```

Comment:
Seems that `Headers` can't be null: https://github.com/microsoft/referencesource/blob/master/System/net/System/Net/Http/HttpContent.cs#L38 (:heart: docs)

Reply:
Nice. I just checked the docs and they didn't tell, so checked all just in case. Considered using an option / nullable builder for this to make it look nicer. Might remove a few steps in a later PR.